src/components/NFTMedia/index.tsx

```diff
@@ -38,22 +39,16 @@ const cx = classNames.bind(styles);
 // Useful because our IPFS links don't have
 // a file extension
 export const checkMediaType = (hash: string) => {
-	return new Promise((resolve, reject) => {
-		fetch('https://gateway.ipfs.io/ipfs/' + hash, { method: 'HEAD' })
-			.then((r: Response) => {
-				const contentTypeHeader = r.headers.get('Content-Type');
-
-				if (contentTypeHeader?.startsWith('image')) {
-					resolve(MediaType.Image);
-				} else if (contentTypeHeader?.startsWith('video')) {
+	return new Promise((resolve) => {
+		fetch(generateCloudinaryUrl(hash, 'video'), { method: 'HEAD' }).then(
```

Comment:
@colbr  Do we intend to check cloudinary instead of ipfs and see if the video is available or not?

Reply:
@padala yep, IPFS has been really patchy, so I think we should check Cloudinary instead. Going to do an extra iteration on this one, this PR is just a hotfix for the videos that aren't loading

Reply:
@colbr yeah ipfs has been patchy.